front/SettingsForm.tsx

```diff
@@ -99,13 +102,15 @@ const SettingsForm = () => {
               className="form-control form-control-lg"
               type="password"
               placeholder="New Password"
+              data-cy="password-settings"
```

Comment:
Consider using a more descriptive name for the data-cy attribute, such as 'password-input'.